src/assets/styles/styles/custom/_variables.scss

```diff
@@ -10,6 +10,8 @@ $enable-print-styles: true !default;
 
 // Color system
 
+$app-font-color: #1d3557 !default;
+$body-color: #fafafa !default;
```

Comment:
Nie wiem czy to najlepsze nazwy:
`app-font-color` - jest używany głównie do napisów, ale nie tylko do napisów
![image](https://user-images.githubusercontent.com/21079319/99926713-dd5ec880-2d42-11eb-9457-3bb8bcf724e0.png)

`body-color` - to kolor tła który jest używany na figmie
![image](https://user-images.githubusercontent.com/21079319/99926997-ef8d3680-2d43-11eb-955a-5c59379412f8.png)

Reply:
Jeśli używany jest innych miejscach to wtedy dodałbym kolejną zmienną, nawet jeśli ma mieć taką samą wartość tak, żeby mozna to było łatwo podmienić, bo np. Aga stwierdzi, że zmienia kolor czcionek. Co do samego nazewnictwa to nazwałbym to primary-text-color i background-color, ale to już jak uważasz.